app/helpers/job_invocations_chart_helper.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal:true
+
+module JobInvocationsChartHelper
+  def job_invocation_chart(invocation)
+    donut_chart('#status_chart',
+                job_invocation_data(invocation)[:columns],
+                job_invocation_data(invocation)[:groups])
+  end
+
+  def job_invocation_data(invocation)
+    success = job_invocation_success_count(invocation)
+    cancelled = job_invocation_cancelled_count(invocation)
+    failed = job_invocation_failed_count(invocation)
```

Comment:
I believe we already have more accurate implementation of getting the counts https://github.com/theforeman/foreman_remote_execution/blob/f0cd7abce067b119d06d1e2bcddbcca7a95bd04c/app/helpers/remote_execution_helper.rb#L16

Perhaps some missed rebase?

Reply:
Definitely, I will implement this. The PR right now can't be merged anyway until https://github.com/theforeman/foreman/pull/5137 is merged too, it's been difficult to keep this PR updated & still making sense ( > 2 years of rebases)

Reply:
Sure, I'm quite confident we're getting to the successful merge soon :)